src/arrays.js

```diff
@@ -1,61 +1,76 @@
 const getNthElement = (index, array) => {
-  // your code here
+  m = (array.length);
```

Comment:
You haven't defined whether your variable `m` is a `const` or a `let`. It is good practice to do so. 
We should always try to name our variable so they describe what they represent. So in this case a better variable name would be `length` or `arrayLength`, rather than using `m`.
You don't need the parenthesis/brackets here around `array.length`. It won't stop the code from running correctly, but it's just unnecessary.